x/twasm/contract/incoming_msgs.go

```diff
@@ -279,6 +281,22 @@ type ConsensusParamsUpdate struct {
 	Evidence *EvidenceParams `json:"evidence,omitempty"`
 }
 
+// Delegate.
+// See
+type Delegate struct {
+	Denom      string `json:"denom"`
+	Amount     string `json:"amount"`
```

Comment:
you can use the wasmvm Coin typ instead of 2 separate fields

Reply:
This comes from `Mint` that also has that structure. Will change both.